Source/controls/modifier_hints.cpp

```diff
@@ -115,6 +131,56 @@ void DrawCircleMenuHint(const Surface &out, const CircleMenuHint &hint, const Po
 	DrawString(out, hint.bottom, origin + Displacement { hint.xMid - hint.bottomW / 2, LineHeight * 2 }, CircleMenuHintTextColor(IsBottomActive(hint)));
 }
 
+/**
+ * @brief Draws hint text for a four button layout with the top/left edge of the bounding box at the position given by origin plus the icon for the spell mapped to that entry.
+ * @param out The output buffer to draw on.
+ * @param hint Struct describing the text to draw and the dimensions of the layout.
+ * @param origin Top left corner of the layout (relative to the output buffer).
+ */
+void DrawSpellsCircleMenuHint(const Surface &out, const CircleMenuHint &hint, const Point &origin)
+{
+	std::optional<CelSprite> pSBkIconCels = LoadCel("Data\\SpellI2.CEL", IconSize);

```

Comment:
This looks like it would cause it to load the graphics every frame, since it's already loaded the global `pSBkIconCels` simply use it instead of shadowing it.

Reply:
done